docs/examples/eggsample/eggsample/host.py

```diff
@@ -0,0 +1,32 @@
+import random
+
+from eggsample import hookimpl, hookspecs, lib, pm
+
+condiments_tray = {"pickled walnuts": 13, "steak sauce": 4, "mushy peas": 2}
+
+class FoodMaster2000:
+    def __init__(self):
+        self.ingredients = []
+
+    def add_ingredients(self):
+        pm.hook.eggsample_add_ingredients(ingredients=self.ingredients)
+
+    def prepare_the_food(self):
+        random.shuffle(self.ingredients)
+
+    def serve_the_food(self):
+        pm.hook.eggsample_prep_condiments(condiments=condiments_tray)
+        print(f"Your food: {', '.join(self.ingredients)}")
+        print(f"Some condiments: {', '.join(condiments_tray.keys())}")
+
+def main():
+    pm.add_hookspecs(hookspecs)
+    pm.load_setuptools_entrypoints("eggsample")
+    pm.register(lib)
+    fm2000 = FoodMaster2000()
```

Comment:
Thought I love the name `FoodMaster2000` definitely seems like one of those cases where you ask *why is this a class and not just functions*?

Reply:
Passed in pluginmanager and did it similar to how real world classes handle it in pytest or tox. Gives the class a reason to exist now (but I couldn't help but chaging the name using your pun :)).